components/export/api/src/main/java/org/phenotips/export/internal/DataToCellConverter.java

```diff
@@ -247,35 +246,53 @@ public DataSection phenotypeBody(Patient patient) throws Exception
         if (sortedFeatures.size() == 0) {
             // offset is included to account for the presence of both "positive" and "negative" in "present"
             int offset = bothTypes ? 1 : 0;
-            Integer emptyX = 0;
             for (int i = 0; i < present.size() - offset; i++) {
-                DataCell cell = new DataCell("", emptyX, 0);
+                DataCell cell = new DataCell("", i, 0);
                 section.addCell(cell);
-                emptyX++;
             }
         }
         // section.finalizeToMatrix();
         return section;
     }
 
+    public void variantsSetup(Set<String> enabledFields) throws Exception
+    {
```

Comment:
I think the optimal solution would be  once the variants branch will be merged to simply  rebase this branch and run it ontop of variants. Thus I wont need to make splits and multiply pull requests.